app/src/main/java/org/stacks/app/ui/account/AccountActivity.kt

```diff
@@ -0,0 +1,58 @@
+package org.stacks.app.ui.account
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.lifecycleScope
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.android.synthetic.main.activity_account.*
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import org.stacks.app.R
+import org.stacks.app.data.IdentityModel
+import org.stacks.app.ui.BaseActivity
+import org.stacks.app.ui.secret.SecretKeyActivity
+import reactivecircus.flowbinding.android.view.clicks
+
+@AndroidEntryPoint
+class AccountActivity : BaseActivity() {
+
+    private val viewModel: AccountViewModel by lazy {
+        ViewModelProvider(this).get(AccountViewModel::class.java)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_account)
+        setNavigation()
+
+        viewModel
+            .identities()
+            .onEach(::setIdentitiesRows)
+            .launchIn(lifecycleScope)
+
+        viewModel
+            .errors()
+            .onEach { finish() }.launchIn(lifecycleScope)
```

Comment:
Line break missing.

Reply:
If we're finishing when there's an error, why isn't the view model just telling the view to finish?